hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -732,6 +740,11 @@ void setListMaxResults(int listMaxResults) {
     this.listMaxResults = listMaxResults;
   }
 
+  @VisibleForTesting
+  void setIsNamespaceEnabledAccount(String isNamespaceEnabledAccount) {
+    this.isNamespaceEnabledAccount = isNamespaceEnabledAccount;
```

Comment:
the method signature should be simpler, it is better to declare it as `boolean` and do the string conversion inside this method, or declare the `isNamespaceEnabledAccount` as `Enum` to differentiate between  `NOT_SET, TRUE and FALSE.`

Reply:
Done